Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_InteractableObject.cs

```diff
@@ -283,13 +294,20 @@ public void SaveCurrentState()
             if (grabbingObject == null)
             {
                 previousParent = transform.parent;
-                previousKinematicState = rb.isKinematic;
+
+                if (rb)
```

Comment:
@mattboy64 yeah i realised that when I read down more :) (shall I just delete these comments?)